website/content/docs/agent-and-proxy/proxy/caching/static-secret-caching.mdx

```diff
@@ -0,0 +1,140 @@
+configuration, described below. The default is to check per token every five minutes. For a setup where the `api_proxy`
+has been configured with `use_auto_auth_token` as `true` or `force`, and all requests going through Proxy will be
+using this sole token, this will be a single request to Vault per this interval, no matter how many secrets are
+being cached.
+
+## Functionality
+
+When enabled, Vault Proxy will cache `GET`s to KVv1 and KVv2 endpoints. When a `GET` is made to a KV secret,
+we will cache the response in Proxy's cache. Once the response is cached, subsequent `GET`s to this secret will
+be served from the cache instead of being forwarded to Vault, as long as this client has access to this secret
+in Vault.
+
+If a token requests a KV secret, we store in the cache that this token has the capability to access a secret.
+Therefore, subsequent requests by the same token will be able to access this secret from the cache. For a second
+token to access the same secret, it must first complete a successful `GET` to the secret before its permission
+to access the secret is cached, and its subsequent requests can be served from the cache instead of Vault itself.
+
+Vault Proxy uses the [event system](/vault/docs/concepts/events) to keep the cache up to date. It will
+pay attention to events that indicate both that the secret is one we have in our cache, and also that
+the secret has been modified (e.g. changed, deleted). Upon receiving such an event, it will update or
+evict the cache entry as appropriate.
+
+Vault Proxy will also intermittently check the permissions of tokens that can access cache entries. This interval
+defaults to once every five minutes, but is configurable by the `static_secret_token_capability_refresh_interval` configuration.
+Every interval, Proxy will perform a call to [`sys/capabilies-self`](/vault/api-docs/system/capabilities-self) on
+each token's behalf to validate that it still has previously-demonstrated capabilities to read secrets from the cache.
+If there is a difference, and a permission (or the token itself) has been revoked, then the cache will be updated,
+and it will no longer be to able access those paths from the cache. Note that the `static_secret_token_capability_refresh_interval`
+configuration can be seen as the maximum period after which permission to read a KV secret can be revoked that a
+token will retain access.
+
+As with other requests to Proxy when the cache is enabled, Proxy will return `HIT` or `MISS` in the `X-Cache`
+response header for requests, to tell the client if it has been served from the cache. The `Age` header
+will be added if it is a cache hit, to inform the client how long it has been in seconds since an update
+to the secret. Note that a long age doesn't necessarily mean that the cache is stale, and could just mean
+that it hasn't been updated in a while.
```

Comment:
```suggestion
When static secrete caching is enabled, Proxy returns `HIT` or `MISS` in the `X-Cache`
response header for requests so client can tell if the response was served from
the cache or forwarded from Vault. In the event of a hit, Proxy also sets the
`Age` header to indicate, in seconds, how old the cache entry is.

<Tip title="Old does not mean stale">

  The fact that a cache entry is old, does not necessarily mean that the
  information is out of date. Vault Proxy continually monitors KV events for
  updates. A large value for `Age` may simply mean that the secret has not been
  rotated recently.

</Tip>
```

Style correction: write in the active voice, avoid contractions. Suggest making the comment about the Age header more explicitly tied to secret hygiene and calling it out as a tip since it's not directly related to Proxy functionality.